Achilles/World.pde

```diff
@@ -0,0 +1,50 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * World represents the entire running world of the game. It should expose functionality
+ * to the base Achilles app so that it can respond to input, update itself, and draw itself
+ */
+class World {
+  private Set<Entity> entities = new HashSet<Entity>();
```

Comment:
I a `Set` object here instead of a `List` because sets don't allow you to add the same element more than once. I've had some reeealllly annoying bugs in other games that happened because somehow more than one reference to the same `Entity` ended up in the world!